client/container_start.go

```diff
@@ -9,7 +9,9 @@ import (
 // ContainerStart sends a request to the docker daemon to start a container.
 func (cli *Client) ContainerStart(ctx context.Context, containerID string, checkpointID string) error {
 	query := url.Values{}
-	query.Set("checkpoint", checkpointID)
+	if len(checkpointID) != 0 {
```

Comment:
OK. So I'll move it to an `Option` struct, I can see that `checkpoint` is not in use in master branch, so I guess this won't break API.